### ISSUE TITLE
Postinstall script fails

### ISSUE THREAD

**Issue by calebegg** [xaverh/vscode-clang-format-provider#10]
There's a postinstall script that attempts to execute `node ./node_modules/vscode/bin/install`, but that file (`install`) does not seem to exist on a clean installation of the vscode package

This causes `npm install` to fail:

```
npm ERR! Failed at the clang-format@0.6.1 postinstall script 'node ./node_modules/vscode/bin/install'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the clang-format package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./node_modules/vscode/bin/install
```

**Comment by martine** [comment]
I have a fix for this, blocked on #13

**Comment by xaverh** [comment]
Fixed in #15
